src/components/LoginForm/LoginForm.tsx

```
@@ -0,0 +1,160 @@
+                  }
+                />
+              </div>
+            );
+          }}
+        />
+
+        <form.Field
+          name="password"
+          children={(field) => {
+            return (
+              <div className={s.fieldWrapper}>
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)